config/consensus.go

```diff
@@ -1126,6 +1126,9 @@ func initConsensusProtocols() {
 
 	vFuture.LogicSigVersion = 7
 
+	// Decrease ledger's max balance lookback. TODO: revisit the value in the future.
+	vFuture.MaxBalLookback = 16;
```

Comment:
I think we should not change `MaxBalLookback` but instead set add a new parameter.
We still need the original `MaxBalLookback` value in online account tracker to calculate expiration. We still need it in the catchpoint tracker to implement PoC for two stage catchpointing with backward compatibility.